bakery/src/tasks/check-feed.js

```diff
@@ -0,0 +1,45 @@
+const dedent = require('dedent')
+
+const task = ({ awsAccessKeyId, awsSecretAccessKey, bucketName, feedFileUrl }) => {
+  return {
+    task: 'check feed',
+    config: {
+      platform: 'linux',
+      image_resource: {
+        type: 'docker-image',
+        source: {
+          repository: 'openstax/cops-bakery-scripts'
+        }
+      },
+      params: {
+        AWS_ACCESS_KEY_ID: `${awsAccessKeyId}`,
+        AWS_SECRET_ACCESS_KEY: `${awsSecretAccessKey}`
+      },
+      outputs: [{ name: 'book' }],
+      run: {
+        path: '/bin/bash',
+        args: [
+          '-cxe',
+          dedent`
+          exec 2> >(tee book/stderr >&2)
+          curl ${feedFileUrl} -o book-feed.json
+          feed_ids=$(cat book-feed.json | jq -r '..|.feed_id?')
+          echo "$feed_ids" | while read item
+          do
+            aws s3api head-object --bucket ${bucketName} --key "$item/.complete" || {
```

Comment:
Will adding a `.complete` file be necessary? Is there a reason that the item's existence in s3 won't give us the same information?

Reply:
Good question. This initial pass is based on the passing comment in our group discussions where we mentioned assuming some type of file marker is dropped at the end of uploads (e.g. in case an upload fails partially), but is one of those definite placeholders that I think will need to get adjusted when this is revisited in [cnx #965](https://github.com/openstax/cnx/issues/965). You can see this is part of some of the opens / questions I left there.